### DISCUSSION TITLE
Does it use proton?

### DISCUSSION THREAD

**Discussion by Seanmclem** [lutris/lutris#4227] [Q&A] [answered]
For example, can I run Gog games on Linux with proton via this tool?

**Comment by strycore** [comment, marked as answer]
Yes, absolutely, we've been able to use Proton even before umu existed but now that we have umu, Proton-GE is the main way to run Windows games in Lutris. We recommend Proton-GE over Valve's Proton since we maintain this more closely. 

Note there are some uneducated cavemen out there that still might be under the impression that Lutris does not support Proton, we wish them a quick recovery and hope they will join us in the modern age at some point in the near future.

**Comment by Seanmclem** [comment]
AFAIK Proton originally was separate from Steam and was just highly optimized for it, and works out of the box with GameHub
